app/models/experimentation/notifications_experiment.rb

```diff
@@ -38,7 +38,8 @@ def self.eligible_patients
           recent_experiment_memberships: Experimentation::TreatmentGroupMembership
                                            .joins(treatment_group: :experiment)
                                            .where("treatment_group_memberships.patient_id = patients.id")
-                                           .where("end_time > ?", RECENT_EXPERIMENT_MEMBERSHIP_BUFFER.ago)
+                                           .where("end_time > ?", Time.now)
```

Comment:
Now that we've introduced this - do we need a test that says "does not include patient in experiment in progress"?